test/extensions/filters/listener/original_src/BUILD

```diff
@@ -49,3 +51,23 @@ envoy_extension_cc_test(
         "@envoy_api//envoy/extensions/filters/listener/original_src/v3:pkg_cc_proto",
     ],
 )
+
+envoy_proto_library(
+    name = "original_src_fuzz_test_proto",
+    srcs = ["original_src_fuzz_test.proto"],
+    deps = [
+        "@envoy_api//envoy/extensions/filters/listener/original_src/v3:pkg",
+    ],
+)
+
+envoy_cc_fuzz_test(
+    name = "original_src_fuzz_test",
+    srcs = ["original_src_fuzz_test.cc"],
+    corpus = "original_src_corpus",
+    deps = [
+        ":original_src_fuzz_test_proto_cc_proto",
+        "//source/extensions/filters/listener/original_src:original_src_lib",
+        "//test/fuzz:utility_lib",
```

Comment:
nit: unneeded dep

Reply:
Changed that dep to //test/mocks/network:network_mocks - more direct.